frontend/src/main/webapp/ViewTrip.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Trip Me Out</title>
+    <script src="ViewTripJavaScript.js"></script>
+  </head>
+  <body onload="setTripPlaceContainerWithServlet()">
+    <h1>Trip Me Out!</h1>
+    <h2> View Trip</h2>
+    <button>Fake Plus Button</button>
+    <button>Fake Minus Button</button>
+    <section id="Trip-Place">
+        <p>Trips will live below</p>
```

Comment:
You probably wan to add a [`<noscript>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/noscript) tag here to indicate that this page requires javascript.